pom.xml

```diff
@@ -33,7 +33,7 @@
   </licenses>
 
   <properties>
-    <jenkins.version>2.0</jenkins.version>
+    <jenkins.version>2.60.3</jenkins.version>
```

Comment:
Does this mean that running the current plugin on jenkins 2.40 would fail?

Reply:
I should work, but the support for Java 8 was only official in `2.60.3`. So, it could work on `2.40`. Also, this version is already pretty old, so it shouldn't be a problem just to align this version.

Reply:
Very well!